bats/sql.bats

```diff
@@ -848,14 +847,49 @@ SQL
   [ $status -ne 0 ]
 }
 
-@test "sql select with json output supports datetime" {
+@test "sql: select with json output supports datetime" {
     run dolt sql -r json -q "select * from has_datetimes"
     [ $status -eq 0 ]
     [[ "$output" =~ "2020-02-17 00:00:00" ]] || false
 }
 
-@test "dolt_version() func" {
+@test "sql: dolt_version() func" {
     SQL=$(dolt sql -q 'select dolt_version() from dual;' -r csv | tail -n 1)
     CLI=$(dolt version | cut -f 3 -d ' ')
     [ "$SQL" == "$CLI" ]
 }
+
+@test "sql: stored procedures sanity check" {
```

Comment:
Need more tests here. At least a test of drop with N>1 procedures stored. And hopefully a test of show as well.